.github/workflows/build.yml

```
@@ -21,11 +21,10 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
-                symfony: ["5.4.*", "^6.0"]
-                sylius: ["^1.12"]
-                node: ["14.x", "16.x", "18.x"]
-                mysql: ["5.7", "8.0"]
+                php: ["8.1", "8.2", "8.3"]
+                symfony: ["^5.4", "^6.4"]
+                node: ["20.x"]
+                mysql: ["8.3"]
```

```suggestion
                mysql: ["8.0"]
```
I would be for testing the LTS version, in the near future, we could probably add 8.4 as the newly released LTS soon